### PR TITLE
Gromm/delete form sections2

### PR DESCRIPTION
**Pull request by guy-ro**
Added functionality to delete form sections. 

Added Greg's fix to the form model.

Still missing undo deletes.

### CODE REVIEW

File: src/Components/FormCreation/CreateEditForm.js [hunk 1]
```diff
@@ -7,12 +7,28 @@ import usePromise from "../../Hooks/usePromise";
 import CreateEditFormHeader from "./CreateEditFormHeader";
 import { defaultFormState } from "./CreateEditFormStateManagement";
 import customFormSectionsReducer from "../../Reducers/CustomFormSectionsReducer";
+import DeleteSectionConfirmation from "./DeleteSectionConfirmation";
```
Comment (GodGreg) [inline src/Components/FormCreation/CreateEditForm.js:10]: remove unused imports

**Review by GodGreg — CHANGES_REQUESTED** [review]
The CSS of the dialog does not match the figma

image
vs 
image

Please see ticket for Figma links. On Figma, you can hover over an element and it will show you its dimensions and the distance is it from other objects. As well the CSS is generated on the right-hand side (which you can pretty much just copy/paste)

**Comment by guy-ro** [comment]
Thanks Greg! Didn't know you could do that with Figma cause it only shows you those tools if you log in.

Updated the style to that shown in Figma.

For the unused imports, I commented out those that I was using in implementing the undo delete feature.

**Comment by GodGreg** [comment]
> Thanks Greg! Didn't know you could do that with Figma cause it only shows you those tools if you log in.
> 
> Updated the style to that shown in Figma.
> 
> For the unused imports, I commented out those that I was using in implementing the undo delete feature. Do you want me to remove them in addition to commenting them out?

Its fine if used for the undo-delete feature

**Review by GodGreg — APPROVED** [review]

**Pull request merged**